docs/computing/connecting/ssh-windows.md

```diff
@@ -116,6 +121,9 @@ generator](https://www.puttygen.com/). The PuTTY documentation provides
 
 The recommended way to copy a public key to a supercomputer is
 [through the MyCSC customer portal](ssh-keys.md#adding-public-key-in-mycsc).
+This will also be the only possible way to upload your key starting April 14,
+2025.
+
 Alternatively, you can do it using PuTTY by following the instructions below.
```

Comment:
All alternative instructions should be removed (this is about supercomputers). 

Do we need those still for e.g. cPouta, there users still of course manage keys. In context of supercomputers all alternatives can be removed.